build.gradle.kts

```
@@ -112,35 +108,37 @@ graal {
 }
 
 dependencies {
-  implementation("io.rsocket.kotlin:rsocket-core-jvm:0.13.0")
-  implementation("io.rsocket.kotlin:rsocket-transport-ktor-client-jvm:0.13.0")
-  implementation("io.rsocket.kotlin:rsocket-transport-ktor-jvm:0.13.0")
-  implementation("io.ktor:ktor-network-tls:1.5.0")
-  implementation("io.ktor:ktor-client-okhttp:1.5.0")
-  implementation("io.ktor:ktor-client-core-jvm:1.5.0")
+  implementation("io.rsocket.kotlin:rsocket-core-jvm:0.15.4")
+  implementation("io.rsocket.kotlin:rsocket-transport-ktor:0.15.4")
+  implementation("io.rsocket.kotlin:rsocket-ktor-client:0.15.4")
+  implementation("io.rsocket.kotlin:rsocket-transport-ktor-websocket-client:0.15.4")
+  implementation("io.rsocket.kotlin:rsocket-transport-ktor-tcp:0.15.4")
+  implementation("io.ktor:ktor-network-tls:2.0.2")
+  implementation("io.ktor:ktor-client-okhttp:2.0.2")
+  implementation("io.ktor:ktor-client-core-jvm:2.0.2")
```

```suggestion
  implementation("io.rsocket.kotlin:rsocket-core:0.15.4")
  implementation("io.rsocket.kotlin:rsocket-transport-ktor-websocket-client:0.15.4")
  implementation("io.rsocket.kotlin:rsocket-transport-ktor-tcp:0.15.4")
  implementation("io.ktor:ktor-network-tls:2.0.2")
  implementation("io.ktor:ktor-client-okhttp:2.0.2")
```

This should be enough, too simplify :)